core/auth/oauth/oidc.go

```diff
@@ -109,6 +109,11 @@ var (
 
 		return ok
 	}
+
+	errNoStateInRequest = errors.New("no state in request")
+	errStateMismatch    = errors.New("state mismatch")
+	errNoIDTokenClaim   = errors.New("claim id_token missing")
+	errGeneric          = errors.New("OpenID Connect error")
```

Comment:
Should we make those exported to be able to handle them?

Reply:
they are only passed to the responder to render the error pages, so there is currently no place where you could check for these errors outside of this package.